src/include/stir/IO/MultiDynamicDiscretisedDensityInputFileFormat.h

```diff
@@ -103,11 +103,11 @@ public InputFileFormat<DynamicDiscretisedDensity>
     }
     // Hard wire some stuff for now (TODO?)
     dyn_disc_den_ptr->set_if_decay_corrected(1.);
-    dyn_disc_den_ptr->set_isotope_halflife(6586.2F);
+    dyn_disc_den_ptr->set_isotope_halflife(dyn_disc_den_ptr->get_exam_info_sptr()->get_radionuclide().get_half_life());
```

Comment:
The test "test_IO_DynamicDiscretisedDensity_Multi" is complaining here. when half_life=-1. 

Also for some reason I don't find this test my STIR version